test/test_data.rb

```diff
@@ -672,5 +672,208 @@ def make_paid_order(params={})
         :charge => make_charge,
       }).merge(params)
     end
+    
+    
+    
+    def country_spec_array
+      {
+        :object => "list",
+        :url => "/v1/country_specs",
+        :data => [
+          make_country_spec,
+          make_country_spec,
+          make_country_spec,
+        ]
+      }
+      
+    end
+    
+    def make_country_spec(params={})
+      {
+        :id=>"US",
+        :object=>"country_spec",
+        :supported_bank_account_currencies=> {:usd => ["US"]},
+        :supported_payment_currencies=>
+        ["usd",
+          "aed",
+          "afn",
+          "all",
+          "amd",
```

Comment:
Let's maybe squash this list of currencies down to four or five items for brevity. I don't think the tests will need them all in there.